### PR TITLE
V4 responsive

### PR DESCRIPTION
**Pull request by jflayhart**
Basically, I just separated the CSS code out and converted into SASS partials.

1. I have the top nav bar responsive, but the menu doesn't work under 480px width because I am waiting on proper implementation of Bootstrap V4. It is ready to work once that is implemented and you can click on your Destiny username and get the menu items as a dropdown.

2. I added CSS min so that the CSS output is minified when running `grunt cssmin`

3. In order to consolidate the top bar area more, I converted some of the textuals to icons. I also removed the gif spinner when loading/refreshing and made the refresh icon spin instead.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,8 @@ build/img
 build/manifest
 build/*.json
 build/manifest.zip
+.DS_Store
+app/.DS_Store
```
Comment (ritcheyer) [inline .gitignore]: maybe change to just `*.DS_store`?

File: app/styles/_layout.scss [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+    bottom: 12px;
+    left: 12px;
+}
+
+.toast-center {
+    top: 45%;
+}
+
+.toast-success {
+	background-color: #51a351;
```
Comment (ritcheyer) [inline app/styles/_layout.scss]: make colors sass variables?

File: app/styles/_objects.scss [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+}
+
+#faq-button {
+    display: inline-block;
+    background-color: #5b6e6c;
+    color: rgba(245, 245, 245, .6);
+    border-radius: 50%;
+    width: 14px;
+    height: 14px;
+    line-height: 14px!important;
```
Comment (ritcheyer) [inline app/styles/_objects.scss]: does this work with no space between `14px` and `!important`

**Comment by ritcheyer** [comment]
:+1:

File: app/styles/_objects.scss (hunk 3 above)
Comment (jflayhart) [inline app/styles/_objects.scss]: Good catch, I happened to not write that CSS but I should have caught it nonetheless. That doesn't mess up the CSS, but it renders the `!important` declaration useless.

File: app/styles/_layout.scss (hunk 2 above)
Comment (jflayhart) [inline app/styles/_layout.scss]: That's the plan. A lot of this is half-baked, good suggestions though keep 'em coming.

File: .gitignore (hunk 1 above)
Comment (jflayhart) [inline .gitignore]: Good point. Will do that instead.

**Pull request merged**